javascript/index.js

```diff
@@ -1,30 +1,57 @@
 const chronometer = new Chronometer();
-
 // get the buttons:
+
 const btnLeft = document.getElementById('btnLeft');
-const btnRight = document.getElementById('btnRight');
+btnLeft.onclick = function () {
+  if (btnLeft.className === "btn start") {
+    chronometer.startClick(printTime);
+    // printTime as a callback
+    // two functions printMinutes and printSeconds
+  } else if (btnLeft.className === "btn stop") {
+    chronometer.stopClick()
+  }
+};
+const btnRight = document.getElementById("btnRight");
+btnRight.onclick = function () {
+  if (btnRight.className === "btn reset") {
+    chronometer.resetClick()
```

Comment:
I added a function to clear the splits on line 54, it should be used to complement the reset action. You'll then also need to display the reset time again by calling `printTime`
```
if (btnRight.classList.contains('reset')) {
    chronometer.resetClick();
    clearSplits();
    printTime();
}